guides/doc-Release_Notes/master.adoc

```diff
@@ -16,6 +16,7 @@ endif::[]
 include::topics/foreman-3.16.0.adoc[leveloffset=+1]
 
 ifdef::katello[]
+include::topics/katello-4.18.1.adoc[leveloffset=+1]
```

Comment:
When I introduced the release notes I intended them to be in chronological order. So I'd list 4.18.1 before Foreman 3.16.0.